src/sentry/api/validators/external_actor.py

```diff
@@ -0,0 +1,34 @@
+from typing import Optional
+
+from rest_framework import serializers
+
+from sentry.models import Organization, OrganizationIntegration
+
+EXTERNAL_ID_MIN_LENGTH = 1
+EXTERNAL_ID_MAX_LENGTH = 64
+
+
+def validate_external_id_option(external_id: Optional[str]) -> Optional[str]:
+    if not external_id:
+        return None
+
+    if not EXTERNAL_ID_MIN_LENGTH <= len(external_id) <= EXTERNAL_ID_MAX_LENGTH:
+        raise serializers.ValidationError(
+            f"external_id has invalid length ({EXTERNAL_ID_MIN_LENGTH}, {EXTERNAL_ID_MAX_LENGTH})"
```

Comment:
This error reads strangely to me, can you show the length of the external_id? Something like:

external_id has invalid length 500. Must be between 1 and 64.

Reply:
that's a great ID